### PR TITLE
fix(fmt): escape characters are getting removed in auto format

### PR DESCRIPTION
**Pull request by skartikey**
This PR is to fix the issue where escape characters are getting removed in auto-format, causing tests to fail

Fixes: #3659

### CODE REVIEW

File: stdlib/contrib/sranka/teams/teams.flux [hunk 1]
```diff
@@ -16,37 +17,46 @@ message = (url, title, text, summary="") => {
...
     // using string body, object cannot be used because '@' is an illegal character in the object property key
-    summary2 = if summary == "" 
-        then text 
-        else summary
-    shortSummary = if strings.strlen(v: summary2) > summaryCutoff 
-        then "${strings.substring(v: summary2, start: 0, end: summaryCutoff)}..."
-        else summary2
+    summary2 = if summary == "" then
+        text
+else
```
Comment (scbrickley) [inline stdlib/contrib/sranka/teams/teams.flux:25]: Is this expected behavior for the formatter? I don't think we should be completely un-indenting `else` statements like this.

Comment (skartikey) [reply stdlib/contrib/sranka/teams/teams.flux:25]: That is correct. we have an issue already created for fixing that. #3656

Comment (scbrickley) [reply stdlib/contrib/sranka/teams/teams.flux:25]: Ah, perfect. Thanks for the clarification :+1:

**Review by scbrickley — APPROVED** [review]
LGTM!

**Pull request merged**